gtsfm/loader/yfcc_imb_loader.py

```diff
@@ -91,8 +92,8 @@ def get_image(self, index: int) -> Image:
 
         return io_utils.load_image(file_name)
 
-    def get_camera_intrinsics(self, index: int) -> Optional[Cal3Bundler]:
-        """Get the camera intrinsics at the given index.
+    def get_camera_intrinsics_full_res(self, index: int) -> Cal3Bundler:
```

Comment:
Optional for the return type.

Reply:
@ayushbaid we don't return None, though. We just raise an error here.